doc/radosgw/multisite.rst

```diff
@@ -344,8 +344,11 @@ Zones that are within a zonegroup replicate all data in order to ensure that
 every zone has the same data. When creating a secondary zone, run the following
 operations on a host identified to serve the secondary zone.
 
-.. note:: To add a tertiary zone, follow the same procedures used for adding a
-   secondary zone. Be sure to specify a different zone name.
+.. note:: To add a second secondary zone (that is, a second non-master zone
```

Comment:
I've wondered if we would want to follow recent practice and s/master/main/ throughout.

Reply:
I thought about this, and while I am committed to the emancipation and flourishing of every person on earth (for whatever that can possibly mean for an antipodal technical writer of limited influence), the diagram says "master" right now and I just want to get this stuff line-edited so that it answers Alexander Patrakov's reasonable questions about what zones and zonegroups and realms and buckets are and how they work together. 

I'm not opposed to removing hierarchical, hegemonic language. But that's a big project, and doing it right means creating new diagrams in many cases (and in this case specifically).